providers/line/line_test.go

```diff
@@ -48,6 +48,18 @@ func Test_SessionFromJSON(t *testing.T) {
 	a.Equal(s.AccessToken, "1234567890")
 }
 
+func Test_SetBotPrompt(t *testing.T) {
+	t.Parallel()
+	a := assert.New(t)
+
+	p := provider()
+	p.SetBotPrompt("normal")
+	session, err := p.BeginAuth("test_state")
+	s := session.(*line.Session)
+	a.NoError(err)
+	a.Contains(s.AuthURL, "&bot_prompt=normal")
```

Comment:
Looks like this bit of the test can be flaky depending on the order of the query params,

```
 --- FAIL: Test_SetBotPrompt (0.00s)
    line_test.go:60: 
        	Error Trace:	line_test.go:60
        	Error:      	"https://access.line.me/oauth2/v2.1/authorize?bot_prompt=normal&client_id=&redirect_uri=%2Ffoo&response_type=code&state=test_state" does not contain "&bot_prompt=normal"
        	Test:       	Test_SetBotPrompt
```

for example, it may create the URL param prefixed with `?` or `&`. It might be better not to check that leading character, and just do,

```go
a.Contains(s.AuthURL, "bot_prompt=normal")
```

Reply:
Thanks for the good catch! fixed to do `a.Contains(s.AuthURL, "bot_prompt=normal")`!